### DISCUSSION TITLE
Automatically select the right sequence when selecting a callsign already replying to CQ

### DISCUSSION THREAD

**Discussion by lasantiagoa** [danleetw/FT8TW#4] [Ideas]
When you manually select a station that is replying to my CQ and "select" "call <callsign>" the app goes into the first sequence. For example
<My Callsign> <Station Callsign> <grid>

My reply should be

<Station Callsign> <My Callsign> <signal strength>

And yet, the reply is always: <Station Callsign> <My Callsign> <grid>

This happens when you manually select a callsign that has already replied. It doesn't happen in auto-tracking mode. 

73 de Luis, HB9HJU

**Comment by danleetw** [comment]
Should be fixed.
